components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/prompt/GeckoPromptDelegate.kt

```diff
@@ -78,6 +79,26 @@ internal class GeckoPromptDelegate(private val geckoEngineSession: GeckoEngineSe
         }
     }
 
+    override fun onAlert(
+        session: GeckoSession?,
+        title: String?,
+        message: String?,
+        callback: AlertCallback?
```

Comment:
It doesn't look like callback can actually be null. You will get some coverage improvements too if you make it non nullable.